torch/_export/serde/schema.py

```diff
@@ -185,12 +185,29 @@ class NamedArgument:
     arg: Argument
 
 
+@dataclass
+class NNModuleFrame:
+    key: str
+    path: str
+    ty: str
+
+
+@dataclass
+class NodeMeta:
+    stack_trace: Optional[str] = None
+    nn_module_stack: Optional[List[NNModuleFrame]] = None
```

Comment:
I think you don't need to make this optional because there's not NodeMeta in any previous versions.